pkg/daemon/ceph/client/subvolumegroup.go

```diff
@@ -55,3 +57,21 @@ func DeleteCephFSSubVolumeGroup(context *clusterd.Context, clusterInfo *ClusterI
 	logger.Infof("successfully deleted cephfs subvolume group %q", volName)
 	return nil
 }
+
+// PinCephFSSubVolumeGroup pin the cephfs subvolume group
+func PinCephFSSubVolumeGroup(context *clusterd.Context, clusterInfo *ClusterInfo, volName string, cephFilesystemSubVolumeGroup *cephv1.CephFilesystemSubVolumeGroup) error {
```

Comment:
This is confusing as the volume name. It's the filesystem name, right? I see in the ceph doc it's called the volName, but let's use rook terms.
```suggestion
func PinCephFSSubVolumeGroup(context *clusterd.Context, clusterInfo *ClusterInfo, fsName string, cephFilesystemSubVolumeGroup *cephv1.CephFilesystemSubVolumeGroup) error {
```

Reply:
It is used in other places in the same file, so I think lets make this consistent, and with follow up pr I can make it more understandable

Reply:
True, for consistency we can leave it